penguin/web/routes.py

```
@@ -3414,6 +3381,20 @@ async def opencode_config_update(
         raise HTTPException(status_code=500, detail="Failed to update config")
 
 
+@router.get("/api/v1/commands")
+async def api_command_list() -> list[dict[str, Any]]:
+    """Return Penguin command metadata for OpenCode-derived TUI clients."""
+    return list_opencode_commands()
+
+
+@router.get("/api/v1/notifications/config")
+async def api_notification_config(
+    core: PenguinCore = Depends(get_core),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)